test_open_file.py

```diff
@@ -0,0 +1,34 @@
+from eventio_kaitai import EventioKaitaiParser, get_payload_address
+from os import path
+from itertools import zip_longest
+
+TESTFILE = "test_files/corsika_iact/one_shower.dat"
+
+
+def test_can_open_file():
+    EventioKaitaiParser.from_file(TESTFILE)
+
+
+def test_file_has_objects_at_expected_position():
```

Comment:
Thinking about it, I think it is not necessary to test the addresses.

It should be good enough to check the expected types, versions and ids of all expected objects. There is basically no way to get a false positive here, if you got those correct, you have the file structure parsed correctly

Reply:
is there any application that needs the addresses?
if we want to provide this as part of the interface, we should probably test it

Reply:
No, it is just used for testing the Reader and debugging